### DISCUSSION TITLE
Keywords from Lightroom Classic are parsed into individual keywords instead of phrases

### DISCUSSION THREAD

**Discussion by arretx** [photoprism/photoprism#4685] [Q&A]
If I have keywords set on photos in Lightroom that have multiple words, when I publish them to the originals folder for Photoprism, they are parsed as individual tags when scanned.

For example: Motor Boat becomes "motor" and "boat" separately.  This means that if I search Photoprism for "Motor Boat", I also retrieve all photos that _were_ actually tagged with just "motor" and just "boat" in Lightroom Classic.

This is most notably a problem with people's names.  Mister Smith becomes "mister" and "smith", which means if I want to search for my "smith and wesson" photos, I end up seeing pictures of Mr. Smith.

Is there a way to fix this?

**Comment by omerdduran** [comment]
Hello. This functionality is not currently available, but it is on the project roadmap. The development of metadata processing and keyword support is being worked on, but we cannot give a precise release date

**Reply by lastzero** [reply]
We will be happy to take another look at this once the upgraded user interface and new viewer are released:

- #3168
- #1307

Also see:

- #1131
